src/IECoreDelight/Renderer.cpp

```diff
@@ -327,18 +325,45 @@ class DelightOutput : public IECore::RefCounted
 				layerName = IECore::CamelCase::join( layerTokens.begin(), layerTokens.end(), IECore::CamelCase::AllExceptFirst);
 			}
 
-			ParameterList layerParams;
+			ParameterList layerParams( output->parameters() );
 
 			layerParams.add( "variablename", variableName );
 			layerParams.add( "variablesource", variableSource );
 			layerParams.add( "layertype", layerType );
 			layerParams.add( "layername", layerName );
 			layerParams.add( { "withalpha", &withAlpha, NSITypeInteger, 0, 1, 0 } );
 
-			const string scalarFormat = this->scalarFormat( output );
-			const string colorProfile = scalarFormat == "float" ? "linear" : "sRGB";
-			layerParams.add( "scalarformat", scalarFormat );
-			layerParams.add( "colorprofile", colorProfile );
+			const double filterSize = parameter<float>( output->parameters(), "filtersize", 3.0 );
+			if( filterSize != 3.0 )
+			{
+				layerParams.add( { "filterwidth", &filterSize, NSITypeDouble, 0, 1, 0 } );
+			}
+
+			const string scalarFormatQuant = this->scalarFormat( output );
+			const string colorProfileQuant = scalarFormatQuant == "float" ? "linear" : "sRGB";
+			const string scalarFormat = parameter<string>( output->parameters(), "scalarformat", "" );
+			if( scalarFormat == "" )
+			{
+				layerParams.add( "scalarformat", scalarFormatQuant );
+				layerParams.add( "colorprofile", colorProfileQuant );
+			}
+
+			const string lightGroup = parameter<string>( output->parameters(), "lightgroup", "" );
+			vector<string> lightGroupTokens;
+			if( lightGroup != "" )
+			{
+				IECore::StringAlgo::tokenize( lightGroup, ' ', lightGroupTokens );
+			}
+
+			const string customDriverName = parameter<string>( output->parameters(), "customdrivername", "" );
+			if( customDriverName == "" )
+			{
+				m_driverHandle = DelightHandle( context, "outputDriver:" + name, ownership, "outputdriver", driverParams );
+			}
+			else
+			{
+				m_driverHandle = DelightHandle( context, "outputDriver:" + customDriverName, ownership, "outputdriver", driverParams );
+			}
```

Comment:
When talking about supporting multi-part for Arnold, I think we've concluded that we wanted to do it by automatically sharing drivers for outputs that had the same filename. Perhaps we could take the same approach here by just keeping a map of filename->driver?